content/docs/user-guide/testing/getting-started/_index.md

```diff
@@ -0,0 +1,233 @@
+
+## GoogleBenchmark
+
+For performance benchmarks of small pieces of C++ code O3DE uses [GoogleBenchmark](https://github.com/google/benchmark/blob/main/docs/index.md), which is quite similar to GoogleTest. However, the definition of failure is a major difference between a normal test and a benchmark. In most tests a pass/fail status is directly evaluated to a boolean state, creating an objective report of success and failure. Benchmarks instead create a subjective performance metric. These metrics are most valuable when periodically recorded, to help detect trends over time. The only objective failure during a benchmark occurs when the code fails to run or crashes.
+
+The steps for configuring a GoogleBenchmark library are identical to the [steps above for GoogleTest libraries](#googletest), with a couple exceptions. The include statement is `#include <benchmark/benchmark.h>` and the cmake helper function is:
+
+```
+ly_add_googlebenchmark(
+    NAME MyModule.Benchmarks
+)
+```
+
+## PyTest
+
+Certain tests are easier to write in a scripting language like Python. The scope of these tests are typically at the [integration level](https://softwaretestingfundamentals.com/integration-testing/), verifying mid-to-high level functionality of workflows. This type of test helps verify system correctness similar to how an end-user will experience the software. Despite that positive aspect, these tests are typically slow and provide less specific failure information. Not all Python-based tests are integration tests: any Python-based tools will prefer unit tests written in Python. For all tests written in Python, O3DE uses [PyTest](https://docs.pytest.org/).
+
+While interpreted languages like Python can have low performance, these tests should not perform heavy computation in Python. Tests should instead focus on coordinating workflows: signalling events and then verifying a response. Leave heavy operations to the code targeted by a test, and perform simple checks in Python test-code.
+
+There are often two Python interpreter instances used during O3DE tests. This can complicate determining which interpreter executes your code, so pay close attention to how the test is defined:
+
+- Test exists outside of **O3DE Editor**
+  - Use in generic tests that launch applications and send external signals.
+  - Helps monitor for application crashes.
+  - Test file names should start with `test_`.
+  - Uses the same Python interpreter that launches when you run `o3de/python/python.sh` or `o3de\python\python.cmd`.
+
+- Test embedded inside O3DE Editor
+  - Use in `EditorPythonBindings` tests which target workflows inside O3DE Editor.
+  - Helps expose editor functionality to a test script.
+  - Test file names should ***not*** include affixes such as `test_` or `_tests`.
+  - PyTest is not used inside the editor, and most PyTest functionality is not available
+    - PyTest is still initiated by the external Python interpreter instance, to handle Editor crashes.
```

Comment:
I'm a bit confused by this part. Overall the first paragraph seems to be about the two Python interpreter instances, though I'm not sure which they are. I also understand that I need to see how the test is defined to determine which interpreter executes my code. So in the following list of points, I expect to learn about the two Python interpreter instances, and the different definitions of tests and what kind of interpreter they use. 

Instead I see that there are two ways a test is defined: outside of O3DE Editor and inside of O3DE Editor, and a list of characteristics of those two types of tests. However, I did learn that tests outside of O3DE Editor "use the same Python interpreter that launches when you run `o3de/python/python.sh` or `o3de\python\python.cmd.`" But still I'm not sure if the list is fitting for the first paragraph introducing it.

Reply:
I am also unsure how to clarify this further, but am open to suggestions. Perhaps a prescriptive list works better than a descriptive list? For now, I'll add links to existing documentation on LyTestTools and EditorTest, though I don't think this fully resolves the complex description.

Worst case, this confusion means the reader is paying attention to the complexity.